Implementation of the buy Shirt feature

dianakrog opened the pull request:
The exception “throws Exception” that is not used in the searchSteps
class is dropped
The annotation AfterAll is changed to After so that it does not affect
the other classes. code is identified
Tefault browser is changed to test with Firefox
The buyShirt feature is created
The buyShirtSteps class is created

DianaGuty committed:
The exception “throws Exception” that is not used in the searchSteps
class is dropped
The annotation AfterAll is changed to After so that it does not affect
the other classes. code is identified
Tefault browser is changed to test with Firefox
The buyShirt feature is created
The buyShirtSteps class is created

DianaGuty committed:
Driver.quit () statements are removed from the steps classes
Adjusted some selectors to improve performance
. Please enter the commit message for your changes. Lines starting

aperdomob commented on src/test/java/Utils/BrowserManager.java:
```diff
@@ -10,7 +10,7 @@
 	public static WebDriver startConfig(){
 
-		switch (System.getProperty("exeBrowser","chrome")) {
+		switch (System.getProperty("exeBrowser","firefox")) {
```
me explicas porque este cambio y que hace esta linea

dianakrog replied:
En la clase BrowserManager se ajusta la declaración de la variable exeBrowser de SystemProperty. Esta es la que se recibe desde el CI para identificar con que Browser se ejecuta la prueba. En el switch por defecto se ejecuta con el navegador que esta en el segundo parametro en caso de que no se le pase nada, para el caso Firefox.

aperdomob commented on src/test/java/stepDefinitions/buyShirtSteps.java:
```diff
@@ -0,0 +1,126 @@
+import io.cucumber.java.After;
+import io.cucumber.java.en.*;
+
+public class buyShirtSteps {
```
las clases en Java empiezan con mayusculas

dianakrog replied:
Se actualiza el nombre de las clases para cumplir el estándar de inicio en Mayúscula. BuyShirtSteps y SearchSteps que tambien tenia este error.

aperdomob reviewed (changes requested):
trata de no usar scroll

DianaGuty committed:
Se actualiza el nombre de las clases para cumplir el estándar de inicio
en Mayúscula.
En la clase BrowserManager se ajusta la declaración de la variable
exeBrowser de SystemProperty. Esta es la que se recibe desde el CI para
identificar con que Browser se ejecuta la prueba
Se agrega el Actions sendKeys para mover hacia el elemento y quitar el
scroll
